projects/plugins/boost/app/lib/Premium_Features.php

```diff
@@ -48,6 +48,13 @@ public static function get_features() {
 		$features = array();
 		// Prepare a list of features after applying jetpack_boost_has_feature_* filter for each feature.
 		foreach ( $all_features as $feature ) {
+			/**
+			 * Filter the feature list to disable some features.
```

Comment:
It's not filtering the feature list, the filter is applies to features individually. So, I think it should say something like: Filter the availability of a feature.

Reply:
Thanks.